packages/marketplace/src/reducers/client/app-detail.ts

```diff
@@ -3,7 +3,11 @@ import { Action } from '@/types/core'
 import { clientFetchAppDetail, clientFetchAppDetailSuccess, clientFetchAppDetailFailed } from '@/actions/client'
 import { isType } from '@/utils/actions'
 
-export type AppDetailData = (AppDetailModel & { apiKey?: string }) | null
+export type AppDetailDataNotNull = AppDetailModel & {
+  apiKey?: string
+}
+
+export type AppDetailData = AppDetailDataNotNull | null
```

Comment:
```
AppDetailData = AppDetailModel & {
  apiKey?: string
} | null
```

Reply:
I intentionally use `AppDetailDataNotNull` because I need 
`
AppDetailModel & {
  apiKey?: string
} 
`

without null stuff since the selector won't return `null`